custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml

```
@@ -200,17 +200,17 @@ card_light:
                 [[[
```

This can be more efficient
```suggestion
                [[[
                  var label = variables.ulm_translation_state
                    if (entity.attributes.brightness != null && entity.state === 'on'){
                      var bri = Math.round(entity.attributes.brightness / 2.55);
                      var label = (bri ? bri : "0") + "%";
                    }
                  return label;
                ]]]
```